### DISCUSSION TITLE
Are LORAs for SDXL actually influencing the output?

### DISCUSSION THREAD

**Discussion by XtremelyBadProgrammer** [vladmandic/sdnext#1909] [Q&A] [answered]
Hi all, just a quick question - I noticed that a recent commit stopped the error message generated when selecting a LORA while using SDXL. Great stuff!

However, the LORAs don't actually seem to apply to the image generation. They look like they are loaded in the output window, but they don't appear to be influencing the generation in any way. I was wondering if they were working for anyone else, if this was a local setting that I somehow missed or if they were not quite completely implemented yet?

Thanks all! And thank you Vlad for all your hard work in supporting SDXL!

**Comment by vladmandic** [comment, marked as answer]
They do, but they also need a bit higher steps count and strength to show up nicely than in SD1. 5
